src/js/veteran-representative/2122-schema.json

```diff
@@ -0,0 +1,91 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Appointment of Veterans Service Organization As Claimant's Representative",
+  "type": "object",
+  "definitions": {
+    "fullName": {
+      "type": "object",
+      "properties": {
+        "first": {
+          "type": "string"
+        },
+        "middle": {
+          "type": "string"
+        },
+        "last": {
+          "type": "string"
+        },
+        "suffix": {
+          "type": "string"
+        }
+      }
+    }
+  },
+  "properties": {
+    "veteranFullName": {
+      "type": "string"
+    },
+    "veteranSSN": {
+      "type": "string"
+    },
+    "vaFileNumber": {
+      "type": "string"
+    },
+    "insuranceNumber": {
+      "type": "string"
+    },
+    "claimantFullName": {
+      "type": "string"
+    },
+    "claimantAddress": {
```

Comment:
We have a common address schema that separates out the fields, so we should switch to using that. It's in vets-json-schema, for reference.

Reply:
Ah! I was on a roll but meant to return to this and use that-- thank you for the reminder!